hazelcast/src/main/java/com/hazelcast/internal/metrics/Probe.java

```diff
@@ -84,4 +85,9 @@
      * @return the ProbeLevel.
      */
     ProbeLevel level() default INFO;
+
+    /**
+     * Measurement unit of a Probe. Not used on member, becomes a part of the key.
+     */
+    ProbeUnit unit() default UNSPECIFIED;
```

Comment:
I would make it count by default.

Reply:
OK, glancing at all the current metrics, maybe 3/4 of them are COUNT, changing